packages/core/package.json

```diff
@@ -6,8 +6,8 @@
     "start": "npm-run-all build -p build:watch storybook:start",
     "storybook:start": "start-storybook -s .storybook/public -p 6006 --config-dir=.storybook",
     "storybook:build": "build-storybook -s .storybook/public -c .storybook -o ../../dist/website/storybook/core --quiet",
-    "lint": "eslint \"**/*.ts\" --ignore-pattern \"dist\" && stylelint \"./**/*.scss\" && lit-analyzer ./src",
-    "lint:fix": "eslint --fix \"./**/*.ts\" && stylelint --fix \"./**/*.scss\"",
+    "lint": "eslint \"**/*.ts\" --ignore-pattern \"dist\" && stylelint \"./**/*.scss\" --ignore-pattern \"tokens.scss\" && lit-analyzer ./src",
+    "lint:fix": "eslint --fix \"./**/*.ts\" && stylelint --fix \"./**/*.scss\" --ignore-pattern \"tokens.scss\"",
```

Comment:
Rather than adding the ignore here I realized we have a ignore option in the style lint config here 
https://github.com/vmware/clarity/blob/next/packages/core/stylelint.config.js#L42

```js
ignoreFiles: ['./src/styles/tokens/generated/*.scss', './dist/**/*.scss'],
```

Reply:
@coryrylan : Per our conversation, I'm resolving this request for now. Plz check back in and confirm.

Reply:
We have `.stylelintrc` file where you can specify the `ignoreFile` patterns.

Reply:
Yes. @coryrylan mentioned that. But in our conversation earlier today, he said he was already working in those build files and 👍 this quick fix for now.

Reply:
the work he is doing is going to change the version of this plugin among other things.